stratum/hal/lib/tdi/tdi.proto

```diff
@@ -0,0 +1,46 @@
+// Copyright 2020-present Open Networking Foundation
+// Copyright 2022 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+
+syntax = "proto3";
+
+option cc_generic_services = false;
+
+package stratum.hal.barefoot;
+
+import "p4/config/v1/p4info.proto";
+
+// This message describes the public Barefoot-specific definition of a single P4
```

Comment:
If TDI is vendor-neutral, shouldn't we replace `Barefoot`, `tofino.bin`, `bfruntime_info`, and `bfrt.json` with generic names?

Reply:
Please remove all references to bfrt, barefoot, tofino, etc.

Reply:
So is this just a renaming exercise? The problem with renaming things is that if you ever want to sync from the upstream Stratum, it will cause a *bunch* of problems. Unless the plan is to never sync from upstream, I would not casually rename things in a fork.